src/test.py

```diff
@@ -0,0 +1,76 @@
+import unittest
```

Comment:
I think the test should probably live in `slack-plays-pokemon/tests/tests_gen_1_pokemon.py` if you could move it there! :D

Reply:
Done. Turns out I needed a lil test infrastructure in `tests/__init__.py` to make imports happy.

Also updated the README with the command to execute them 🙂